### DISCUSSION TITLE
Is there any detailed documentation for Git Gui?

### DISCUSSION THREAD

**Discussion by ScreaminScott** [git-for-windows/git#4441] [Q&A] [answered]
I was looking for some detailed documentation for the Git Gui tool.  Does it exist anywhere?
The Git Gui Help only takes you to documentation for git.  That is useful if you are using the command line.  
But it would be helpful to have documentation for the gui that explains all of the screens and the options on each.

**Comment by dscho** [comment]
This is not a Windows-specific issue. As such, I bet you'll have more luck getting good answers on [the Git mailing list](mailto:git@vger.kernel.org) (send plain-text messages, HTML messages are dropped silently).

**Comment by ccalvocm** [comment]
Greetings @ScreaminScott 

Yes, there is documentation available for Git Gui that explains all of the screens and options. Here are a few resources that you can use to learn more about Git Gui:

Git Gui User Manual: This is the official user manual for Git Gui, and it provides detailed information on how to use the tool. You can access it by running the command git help gui in your terminal or by visiting the following link: [https://git-scm.com/docs/git-gui](https://git-scm.com/docs/git-gui)

Git Gui Tutorial: This is a step-by-step tutorial that walks you through the process of using Git Gui to manage your Git repositories. It covers everything from creating a new repository to committing changes and pushing them to a remote repository. You can access it by visiting the following link: [https://medium.com/@pawanpiumal1/basics-of-git-gui-71366702fc26](https://medium.com/@pawanpiumal1/basics-of-git-gui-71366702fc26)

**Comment by ScreaminScott** [comment]
The link https://git-scm.com/docs/git-gui is NOT a git gui manual. It is a
git manual. It doesn't have any screen shots of the git gui interface. It
only lists command line instructions.

I haven't had a chance to look at the other links yet.

**Comment by ScreaminScott** [comment]
The tutorial and cheat sheet links are invalid.

So I guess I am assuming that there is absolutely no documentation for the
git gui tool?

**Comment by ccalvocm** [comment]
@ScreaminScott 

Links are working and indeed there was a pictured git gui tutorial. 

How about using GitHub Desktop which has a nice gui?

Tutorial: https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/getting-started-with-github-desktop

**Comment by ScreaminScott** [comment]
Sorry Both the tutorial and cheat sheet links are not valid:
https://imgur.com/a/YWVH6DE

Thanks for the GitHub desktop tutorial. However, I was looking for documentation for the GitGui tool that comes packaged with Git for Windows.

**Comment by ccalvocm** [comment]
@ScreaminScott 

Have you checked this resources?

http://wiki.seas.harvard.edu/geos-chem/index.php/Using_Git_GUI
https://cs.hofstra.edu/docs/pages/guides/git_win_install.html
https://www.red-gate.com/simple-talk/devops/tools/git-in-action/
https://matthew-brett.github.io/pydagogue/git_gui_windows.html
https://medium.com/@pawanpiumal1/basics-of-git-gui-71366702fc26
https://www.geeksforgeeks.org/working-on-git-for-gui/
http://codeur-pro.fr/git-gui-guide-complet/

If none of them serve as a documentation, as the latest version of Git GUI as of August 2021 is version 0.23.0, which was released in May 2021, I would say there's no detailed documentation anymore.

**Comment by ScreaminScott** [comment]
That first link looks great!  It's an older version of Git Gui but it may
be helpful!

I'll check the other ones.  Thanks.

Still bummed there isn't anything official though.

**Comment by ccalvocm** [comment, marked as answer]
Greetings @ScreaminScott 

I contacted git support asking for an official, detailed documentation for git-gui. I got back the documentation link I already posted in https://github.com/git-for-windows/git/discussions/4441#discussioncomment-5983578, so answering the OP @ScreaminScott that is the official-detailed documentation, nevertheless there exists other documented tutorials as I posted in https://github.com/git-for-windows/git/discussions/4441#discussioncomment-6000617.

E-mail received:

```
Hi

On Fri, May 26, 2023, at 16:29, allyriah wrote:
> Greetings:
> Please, I need to know if there's an official documentation for Git Gui.
>
>
> Many thanks in advance,
> Kind regards.

https://git-scm.com/docs/git-gui
```

**Reply by ccalvocm** [reply]
@ScreaminScott
